test/helper.js

```diff
@@ -21,14 +21,44 @@ const purgeDlqBySubcriptionId = async subscriptionId => {
 	await bus.processDlq(subscriptionId, accept);
 };
 
+const purgeActiveBySubcriptionId = async (subscriptionId, n) => {
+	let activeBodies;
+	try {
+		activeBodies = await bus.peekActive(subscriptionId, n);
+	} catch (error) {
+		console.log(error); // eslint-disable-line no-console
+	}
+	const processActiveMessages = async () => new Promise((resolve, reject) => {
+		const timeout = setTimeout(() => {
+			reject();
+		}, 5000);
+
+		let count = 0;
+		const processMessage = () => {
+			count++;
+			if (count === activeBodies.length) {
+				clearTimeout(timeout);
+				resolve();
+			}
+		};
+		const subscribe = () => bus.subscribe(console.error, console.log); // eslint-disable-line no-console
+		subscribe()(subscriptionId, processMessage);
+	});
+	debug(`Peeked ${activeBodies.length} messages in DLQ of ${subscriptionId}`);
```

Comment:
this is not a dead letter queue, please change the message to 

`debug(`Peeked ${activeMessages.length} messages in subscriptionId ${subscriptionId}`);`

or similar

Reply:
Rewriting it now to: